cumulusci/robotframework/tests/cumulusci/base.robot

```diff
@@ -64,6 +64,51 @@ Test Perf - Parser does not choke with no keywords - Should Fail
 Test Perf Measure Other Metric
     Set Test Metric    Max_CPU_Percent    30
 
-Test Should Generate An Error - Mismatched End Perf Timer - Should Fail
-    [Tags]      noncritical
-    End Perf Timer
+Mismatched End Perf Timer - Should Fail
+    Run Keyword and Expect Error
+    ...         *Elapsed time clock was not*
+    ...         End Perf Timer
+
+Test Elapsed Time For Last Record
+    # This test uses contacts as if they were "jobs" becaues they are
```

Comment:
I'd rather see us using the [Documentation] feature for documenting tests. It's not a show-stopper,  though.

Reply:
This isn't really documentation of what the test is testing, it's just documentation for someone reading the code about why it uses a non-intuitive sobject.